float32 x_error
float32 y_error
float32 z_error
float32 roll_error
float32 pitch_error
float32 yaw_error